terracumber_config/tf_files/SUSEManager-4.3-NUE.tf

```diff
@@ -0,0 +1,224 @@
+}
+
+// sumaform specific variables
+variable "SCC_USER" {
+  type = string
+}
+
+variable "SCC_PASSWORD" {
+  type = string
+}
+
+variable "GIT_USER" {
+  type = string
+  default = null // Not needed for master, as it is public
+}
+
+variable "GIT_PASSWORD" {
+  type = string
+  default = null // Not needed for master, as it is public
+}
+
+terraform {
+  required_version = "1.0.10"
+  required_providers {
+    libvirt = {
+      source = "dmacvicar/libvirt"
+      version = "0.6.3"
+    }
+  }
+}
+
+provider "libvirt" {
+  uri = "qemu+tcp://ramrod.mgr.suse.de/system"
+}
+
+module "cucumber_testsuite" {
+  source = "./modules/cucumber_testsuite"
+
+  product_version = "4.3-nightly"
+
+  // Cucumber repository configuration for the controller
+  git_username = var.GIT_USER
+  git_password = var.GIT_PASSWORD
+  git_repo     = var.CUCUMBER_GITREPO
+  branch       = var.CUCUMBER_BRANCH
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+
+  images = ["centos7o", "opensuse152o", "sles15sp2o", "sles15sp3o", "sles15sp4o", "ubuntu2004o"]
+
+  use_avahi    = false
+  name_prefix  = "suma-43-"
+  domain       = "mgr.suse.de"
+  from_email   = "root@suse.de"
+
+  no_auth_registry = "registry.mgr.suse.de"
+  auth_registry = "registry.mgr.suse.de:5000/cucutest"
+  auth_registry_username = "cucutest"
+  auth_registry_password = "cucusecret"
+  git_profiles_repo = "https://github.com/uyuni-project/uyuni.git#:testsuite/features/profiles/internal_nue"
+
+  server_http_proxy = "http-proxy.mgr.suse.de:3128"
+
+  host_settings = {
+    controller = {
+      provider_settings = {
+        mac = "aa:b2:93:01:00:90"
+      }
+    }
+    server = {
+      provider_settings = {
+        mac = "aa:b2:93:01:00:91"
+      }
+    }
+    proxy = {
+      provider_settings = {
+        mac = "aa:b2:93:01:00:92"
+      }
+      additional_packages = [ "venv-salt-minion" ]
+      install_salt_bundle = true
+    }
+    suse-client = {
+      image = "sles15sp2o"
+      name = "cli-sles15"
+      provider_settings = {
+        mac = "aa:b2:93:01:00:94"
+      }
+      additional_packages = [ "venv-salt-minion" ]
+      install_salt_bundle = true
+    }
+    suse-minion = {
+      image = "sles15sp2o"
+      name = "min-sles15"
+      provider_settings = {
+        mac = "aa:b2:93:01:00:96"
+      }
+      additional_packages = [ "venv-salt-minion" ]
+      install_salt_bundle = true
+    }
+    suse-sshminion = {
+      image = "sles15sp2o"
+      name = "minssh-sles15"
+      provider_settings = {
+        mac = "aa:b2:93:01:00:98"
+      }
+      additional_packages = [ "venv-salt-minion", "iptables" ]
+      install_salt_bundle = true
+    }
+    redhat-minion = {
+      image = "centos7o"
+      provider_settings = {
+        mac = "aa:b2:93:01:00:99"
+        // Since start of May we have problems with the instance not booting after a restart if there is only a CPU and only 1024Mb for RAM
+        // Also, openscap cannot run with less than 1.25 GB of RAM
```

Comment:
It's interesting that for 4.2 we have:
```
        // Openscap cannot run with less than 1.25 GB of RAM
        memory = 1280
```
And seems to be working.

Anyway, this is copied from Head.